pkg/scheduler/framework/runtime/framework_test.go

```diff
@@ -1509,8 +1509,8 @@ func TestRunPreFilterPlugins(t *testing.T) {
 					inj:  injectedResult{PreFilterStatus: int(framework.Error)},
 				},
 			},
-			wantPreFilterResult: nil,
-			wantStatusCode:      framework.Error,
+			wantSkippedPlugins: sets.New("skip"),
+			wantStatusCode:     framework.Error,
```

Comment:
add a test case for a Skip, followed by Unschedulable

Reply:
Default plugins exercised the path already.